gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=3.1.1-SNAPSHOT
+VERSION_NAME=3.1.2-SNAPSHOT
```

Comment:
Increment MINOR version number since we are adding backward compatible functionality